mne/viz/topo.py

```diff
@@ -724,11 +718,17 @@ def _plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
         nirs_types = {'hbo', 'hbr', 'fnirs_cw_amplitude', 'fnirs_od'}
         is_nirs = len(set.intersection(types_used, nirs_types)) > 0
         if is_meg:
-            types_used = list(types_used)[::-1]  # -> restore kwarg order
+            types_used = list(types_used)
+            # Fix issue where ylimits get swapped
```

Comment:
Could you add an XXX here, so we can get to the bottom of this at a later point?

```suggestion
            # XXX: Fix issue where ylimits get swapped
```

the randomness is still weird. I suspect it's because of `set` and `dict` above but not sure.